test/services/account/services/accounts.test.js

```diff
@@ -255,7 +255,7 @@ describe('Account Service', () => {
 		it('should fail on verification mismatch', async () => {
 			const user = await testObjects.createTestUser({ firstLogin: true, roles: ['student'] });
 			const accountDetails = {
-				username: 'some@user.de',
+				username: `some${new ObjectId()}@user.de`,
```

Comment:
Should we clarify why old data was not removed?

Reply:
good point, removal is another good way to solve it!

Reply:
the objects have not been added to be cleaned up. this was the quickest change as this relates not to this pr. however adding element ids to the internal lists of items to be cleaned up adding a random value was implemented instead.